hyperapp.d.ts

```diff
@@ -0,0 +1,40 @@
+export as namespace hyperapp
+
+// VirtualNode part
+
+export interface VirtualNode<Data> {
+  tag: string
+  data?: Data
+  children: VirtualElement<{}>[]
+}
+
+export type VirtualElement<Data> = VirtualNode<Data> | string
+
+export interface Component<Data> {
+  (data?: Data, ...children: VirtualElement<{}>[]): VirtualNode<Data>
+}
+
+export function h<Data>(
+  tag: Component<Data> | string,
+  data?: Data,
+  children?: VirtualElement<{}>[] | VirtualElement<{}> | number
```

Comment:
@andrewiggins Is the function info in vscode intellisense picked up from JSDoc comments?